src/Test/L1/Worker/WorkerL1Tests.cs

```diff
@@ -261,6 +263,128 @@ public async Task SignatureEnforcementMode_FailsWhenTasksArentSigned()
             }
         }
 
+        [Fact]
+        [Trait("Level", "L1")]
+        [Trait("Category", "Worker")]
+        public async Task TrackingConfigs_should_be_consistent_across_runs()
```

Comment:
nit: the code doesn't appear to use snake case anywhere else. Following the existing convention this would be `TrackingConfigs_ShouldBeConsistentAcrossRuns`

Reply:
That's fair. Personally, I don't think we name our tests well any where in the product. I.e. the test name should say what the test is testing. But being consistent is important. I will get rid of the snake_case.